src/types/sequencing.ts

```diff
@@ -7,22 +7,28 @@ export enum DictionaryTypes {
   'telemetry_dictionary' = 'telemetry_dictionary',
 }
 
-export type ChannelDictionary = DictionaryType;
+export type ChannelDictionary = {
+  type: 'CHANNEL';
+} & DictionaryType;
 
 export type CommandDictionary = {
-  command_types_typescript_path: string;
+  type: 'COMMAND';
 } & DictionaryType;
 
-export type ParameterDictionary = DictionaryType;
+export type ParameterDictionary = {
+  type: 'PARAMETER';
```

Comment:
These should reference the `enum` that you make.

Reply:
No problem, I added that